rust_snuba/Cargo.toml

```diff
@@ -37,10 +37,12 @@ rust_arroyo = { path = "./rust_arroyo" }
 sentry = { version = "0.32.0", features = ["anyhow", "tracing"] }
 sentry-kafka-schemas = "0.1.40"
 serde = { version = "1.0", features = ["derive"] }
-serde_json = { version = "1.0" }
+serde_json = { version = "1.0", features = ["raw_value"] }
 thiserror = "1.0"
 tokio = { version = "1.19.2", features = ["full"] }
-statsdproxy = { version = "0.1.0", features = ["cadence-adapter"], git = "https://github.com/getsentry/statsdproxy", default-features = false }
+statsdproxy = { version = "0.1.0", features = [
```

Comment:
can you revert the formatting changes here? I feel that between various people, this file gets reformatted a lot.